db/schema.rb

```diff
@@ -494,6 +494,16 @@
 
   add_index "schedules", ["facility_id"], :name => "i_schedules_facility_id"
 
+  create_table "splits", :force => true do |t|
+    t.integer "parent_split_account_id",                               :null => false
+    t.integer "subaccount_id",                                         :null => false
+    t.decimal "percent",                 :precision => 6, :scale => 3, :null => false
+    t.boolean "extra_penny",                                           :null => false
+  end
+
+  add_index "splits", ["parent_split_account_id"], :name => "index_splits_on_parent_split_account_id"
```

Comment:
We have foreigner installed in this project, so we've been doing foreign keys as well as indexes when possible to keep referential integrity. We might want to add those here as well (I'd be fine if that's a separate PR).